README.md

```diff
@@ -96,14 +107,20 @@ To perform migrations manually (without restarting the container) run:
 yarn run web rails db:migrate
 ```
 
-#### Seeding the database
+#### Seeding the database
```

Comment:
What on earth is the difference 😆

Reply:
@Caleb-T-Owens Apparently I replaced a Unicode `&#160;` nobreak space with a normal ASCII space